x-pack/legacy/plugins/monitoring/server/telemetry_collection/get_cluster_uuids.ts

```diff
@@ -5,16 +5,16 @@
  */
 
 import { get } from 'lodash';
-// @ts-ignore
-import { createQuery } from './create_query';
-// @ts-ignore
-import { INDEX_PATTERN_ELASTICSEARCH } from '../../common/constants';
 
 import {
   ClusterDetailsGetter,
   StatsCollectionConfig,
   ClusterDetails,
-} from '../../../../../../src/legacy/core_plugins/telemetry/server/collection_manager';
+} from 'src/legacy/core_plugins/telemetry/server/collection_manager';
```

Comment:
as I understand this is handled by built-in ts functionality. could you add an example for js code to make sure that server runs?